src/main/scala/cromwell/binding/values/WdlString.scala

```diff
@@ -4,7 +4,13 @@ import cromwell.binding.types.WdlStringType
 import org.apache.commons.lang3.StringEscapeUtils
 import scala.util.{Success, Try}
 
-case class WdlString(value: String) extends WdlPrimitive {
+trait WdlStringLike {
+  def value: String
+  def isGcsUrl: Boolean = value.startsWith("gs://")
```

Comment:
Yeah, let's maybe discuss in person... I see what you mean, though I'm not sure what the right solution is.  Currently as it is, they will not get "re-GCSified"